code/datums/components/butchering.dm

```diff
@@ -33,7 +33,7 @@
 
 	if(user.a_intent != INTENT_HARM)
 		return
-	if(M.stat == DEAD && (M.butcher_results || M.guaranteed_butcher_results)) //can we butcher it?
+	if(M.stat == DEAD && (M.butcher_results || M.guaranteed_butcher_results || istype(source, /obj/item/silkknife))) //can we butcher it?
```

Comment:
Yeah so basically what I said before, this should be a new component instead of a subtype of butchering because this technically allows the silkknife to butcher... anything.